Tests/ClientTests/Coordinators/Mocks/MockBrowserCoordinator.swift

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import WebKit
+import Storage
+@testable import Client
+
+class MockBrowserCoordinator: BrowserNavigationHandler, ParentCoordinatorDelegate {
+    var showSettingsCalled = 0
+    var showFakespotCalled = 0
+    var showCreditCardAutofillCalled = 0
+    var showRequiredPassCodeCalled = 0
+    var showLibraryCalled = 0
+    var showHomepanelSectionCalled = 0
+    var showEnchanedTrackingProtectionCalled = 0
```

Comment:
```suggestion
    var showEnhancedTrackingProtectionCalled = 0
```

Reply:
@lmarceau thank you. i'll modify it just now